json-endpoints.php

```diff
@@ -2976,6 +2976,7 @@
 		'jetpack_relatedposts_enabled' => '(bool) Enable related posts?',
 		'jetpack_relatedposts_show_headline' => '(bool) Show headline in related posts?',
 		'jetpack_relatedposts_show_thumbnails' => '(bool) Show thumbnails in related posts?',
+		'jetpack_protect_whitelist' => '(array) List of IP addresses to whitelist',
```

Comment:
(trivial) needs a couple spaces to match alignment with prior aligned lines (neglecting ones that run too long)